src/app/inventory/get-badge-info.ts

```diff
@@ -36,10 +38,21 @@ function processBounty(item: DimItem) {
 }
 
 function processStackable(item: DimItem) {
+  const isCapped =
+    item.amount === item.maxStackSize &&
+    Boolean(item.stackUniqueLabel && item.stackUniqueLabel.length);
   return {
     showBadge: true,
-    badgeClassNames: { 'item-stat': true, 'item-stackable-max': item.amount === item.maxStackSize },
-    badgeCount: item.amount.toString()
+    badgeClassNames: {
+      'item-stat': true,
+      'item-stackable-max': item.amount === item.maxStackSize,
+      'badge-capped': isCapped
+    },
+    badgeCount:
+      item.maxStackSize > 0 && item.stackUniqueLabel && item.stackUniqueLabel.length > 0
+        ? `${item.amount}/${item.maxStackSize}`
+        : item.amount.toString(),
```

Comment:
1. We should probably hide the amount if it's == 1, like in game.
2. Why are we checking `stackUniqueLabel`?

Reply:
1 - makes sense
2 - the presence of stack unique label is how they indicate that a stack is a unique one in the API. There's no boolean or anything.

Reply:
I see, makes sense. We care that it’s unique because it won’t just start another stack.